statchatexpress/app.js

```diff
@@ -3,12 +3,30 @@ import cors from "cors";
 import morgan from "morgan";
 import { sequelize } from "./database.js";
 import { User, Stats } from "./models/index.js";
+import session from "express-session";
 
 const app = express();
-
-app.use(cors());
+const userSecret = process.env.SESSION_SECRET;
+const maximumAge = parseInt(process.env.SESSION_MAX_AGE);
+app.use(
+  cors({
+    origin: "http://localhost:3000",
+    credentials: true,
+  })
+);
 app.use(express.json()); // Middleware for parsing JSON bodies from HTTP requests
 app.use(morgan("combined"));
+app.use(
+  session({
+    secret: userSecret,
+    saveUninitialized: true,
+    resave: false,
+    cookie: {
+      httpOnly: false,
```

Comment:
Can you add a comment on why this is set to false? I know we walked through this today but someone else who comes across this might be confused.